saas/mixins.py

```diff
@@ -1215,7 +1216,12 @@ def get_object(self):
 
 class BalancesDueMixin(DateRangeContextMixin, ProviderMixin):
 
-    filter_backends = (DateRangeFilter,) # XXX Why no search and ordering?
+    search_fields = (
+        'slug',
+        'full_name',
+    )
+
+    filter_backends = (DateRangeFilter, SearchFilter, OrderingFilter)
```

Comment:
There is an `OrderingFilter` but there does not seem to be a default ordering provided.